packages/pg/lib/connection-parameters.js

```diff
@@ -89,6 +111,23 @@ class ConnectionParameters {
     if (this.ssl === 'no-verify') {
       this.ssl = { rejectUnauthorized: false }
     }
+
+    // Fill in possibly missing SSL config parameters from environment variables. 
+    // This lets you mix SSL definitions between connection string, config object and environment variables.
+    if (this.ssl === true) {
+      var sslCandidate = fillInSSLVariablesFromEnvironment({})
+      
+      // Change the type of this.ssl from boolean to object only if relevant environment variables were defined
+      if (!_.isEmpty(sslCandidate)) {
+        this.ssl = sslCandidate
+      }
+    } else if (typeof this.ssl === 'object') {
+      var sslCandidate = fillInSSLVariablesFromEnvironment({})
+      this.ssl = Object.assign(this.ssl, sslCandidate)
```

Comment:
Potentially modifies user-provided `config.ssl` object in place.

```suggestion
      this.ssl = {...this.ssl, ...sslCandidate}
```